infra/src/test/kotlin/fi/fta/geoviite/infra/geometry/DataProductsTestData.kt

```diff
@@ -0,0 +1,36 @@
+package fi.fta.geoviite.infra.geometry
```

Comment:
Eriytetty elementtiluettelon testeistä, sillä näitä tarvitaan myös vaakageometrian testeissä

Reply:
Elementtien luonnille on jo GeometryDomainTestData.kt:ssa moninaisia variantteja. Näistä voisi karsia duplikaatteja ja viedä kaikki samaan paikkaan.